pkgs/tools/video/nordlicht/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, ffmpeg, popt, help2man }:
+    with cmake;
+    stdenv.mkDerivation rec {
+        name = "nordlicht-${version}";
+        version = "0.4.5-git.7ad3f008.4b1751bd";
```

Comment:
please use an official release, if you can't use it for some reason (necessary patches), then please reference the date of the commit:
`unstable-YYYY-MM-DD`

Reply:
if that's the case, then it sould be `rev = version` in the src attr set

Reply:
The tag starts with v: `rev = "v${version}`